src/Tribe/Admin/Help_Page.php

```diff
@@ -66,6 +66,32 @@ public function is_current_page() {
 		return in_array( $current_screen->id, $help_pages );
```

Comment:
I know you didn't touch this method, but it's the same problem as the one below, add the checks here too.

Reply:
This function was improved with the `is_screen()` validation